instrumentation/log4j/log4j-2-common/testing/build.gradle.kts

```diff
@@ -5,6 +5,8 @@ plugins {
 dependencies {
   api(project(":testing-common"))
 
+  api("io.opentelemetry:opentelemetry-sdk-logs:1.9.0-alpha")
```

Comment:
Don't think we need version here since we apply BOM for io.opentelemetry. Similar for all usages

Reply:
Yes, I'm just waiting for this project to update its BOM dependency on 1.9.0 and then will drop the version here.